modules/system/config-wfrp4e.js

```diff
@@ -1697,13 +1701,11 @@ WFRP4E.conditionScripts = {
         let effect = actor.hasCondition("ablaze")
         let value = effect.conditionValue;
 
-        let leastProtectedLoc;
         let leastProtectedValue = 999;
         for (let loc in actor.status.armour)
         {
             if (actor.status.armour[loc].value != undefined && actor.status.armour[loc].value < leastProtectedValue)
             {
-                leastProtectedLoc = loc;
```

Comment:
Revert this change as `leastProtectedLoc` is now used, see https://github.com/moo-man/WFRP4e-FoundryVTT/commit/5781681e737255ea91199f2b4e9e3dbe5113e0a6

Reply:
reversed